tests/rpc-tests/rpc-compat.js

```diff
@@ -0,0 +1,199 @@
+import fs from 'fs'
+import path from 'path'
+import { maxRetries, runScpCommand, runSshCommand, runSshCommandWithReturn } from '../../src/express/common/remote-worker'
+import assert from 'assert'
+import { checkAndReturnVMIndex, loadDevnetConfig } from '../../src/express/common/config-utils'
+import HDWalletProvider from '@truffle/hdwallet-provider'
+import { fundAccount } from '../test-eip-1559'
```

Comment:
Since we are using this as a utility, can you move it to a different file/folder?